underfs/swift/src/main/java/alluxio/underfs/swift/http/SwiftDirectClient.java

```diff
@@ -67,10 +68,10 @@ public static SwiftOutputStream put(Access access, String objectName) {
         httpCon.connect();
         return new SwiftOutputStream(httpCon);
       }
-      LOG.debug("Not an instance of HTTP URL Connection");
-    } catch (IOException e) {
+      throw new Exception("Not an instance of HTTP URL Connection");
+    } catch (Exception e) {
```

Comment:
When does `url.openConnection()` return a non HTTPURLConnection? If it is unexpected, we can just do a precondition check and throw the unchecked exception. In addition, since we are throwing the exception now and not converting it to null, I do not think we need to catch, log, then throw again.